frontend/src/SignUp.jsx

```diff
@@ -0,0 +1,71 @@
+import InputBox from "./components/InputBox";
+import { useNavigate } from "react-router-dom";
+
+const SignUp = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="flex flex-col">
+      <header>
+        <div className="text-center text-8xl m-20 font-bold text-indigo-50 text-shadow-lg">
+          Alpha-Edge
+        </div>
+      </header>
+      <form className="flex flex-col bg-indigo-50 p-8 rounded-md shadow-xl/40 shadow-slate-900 w-150 m-auto">
+        <h3 className="font-bold text-3xl p-1 text-center">Sign Up</h3>
+        <p className="font-bold text-sm p-3 text-center">
+          Please sign up to continue{" "}
+        </p>
+        <InputBox
+          placeholder={"ex: Santiago Criado"}
+          label={"Name"}
+          id={"name"}
+        />
+        <InputBox
+          placeholder={"ex: 1234"}
+          label={"Set Password"}
+          id={"new-password"}
+        />
+        <InputBox
+          placeholder={"ex: Cool-Dude-1"}
+          label={"Username"}
+          id={"name"}
+        />
+        <InputBox
+          placeholder={"bobby@gmail.com"}
```

Comment:
What's the utility of having a name, username, and email here? Usually I see sites using one of username OR email to uniquely identify an account. Are you planning to use each bit of information for a separate purpose?